### ISSUE TITLE
Iframe on /marketing/sharing-buttons needs an accessible name

### ISSUE THREAD

**Issue by youbrokesomething** [Automattic/wp-calypso#75601]
### Quick summary

There's an iframe on /marketing/sharing-buttons that needs an accessible name or the role "presentation" if it isn't supposed to be visible or interacted with by accessibility assisting devices like screen readers.
The iframe is:
`<iframe src="//widgets.wp.com/sharing-buttons-preview/?service%5B%5D%5B0%5D=press-this&amp;service%5B%5D%5B1%5D=twitter&amp;service%5B%5D%5B2%5D=facebook&amp;service%5B%5D%5B3%5D=linkedin" width="100%" frameborder="0" class="official-preview" height="30"></iframe>`

### Steps to reproduce

1. On /marketing/sharing-buttons
2. Inspect the iframe referenced in the summary or use an accessibility checker tool like axe DevTools Chrome extension.

### What you expected to happen

All iframes should have an accessible name or a role that tells screen readers how to handle that content.

### What actually happened

One iframe doesn't have an accessible name.

### Impact

Some (< 50%)

### Available workarounds?

No but the platform is still usable

**Comment by katinthehatsite** [comment]
Hi @youbrokesomething - thanks for reporting the issue. I started looking into this but was not yet able to find the `iframe` in question. This is what am I getting when testing `https://wordpress.com/marketing/sharing-buttons/{site}` page with axe DevTools:

https://github.com/Automattic/wp-calypso/assets/25575134/f1c30cbf-29e6-4937-9058-3ec5c56a0b21

I was also not able to see it when I was inspecting the page. Would you be able to give me a pointer on how to trigger the `iframe` in question to appear or if perhaps the steps that I am following are not correct? Thanks!

**Comment by youbrokesomething** [comment]
The iframe is coming up for me under the parent div .sharing-buttons-preview-button, and the iframe has the class .official-preview. It's part of the Share preview, but your preview looks different than mine... I also have an option for 'Comment Likes: On for all posts' that I don't see on your screen.

![Screen Shot 2023-05-16 at 1 17 51 PM](https://github.com/Automattic/wp-calypso/assets/11060356/bc2fa687-65b6-4273-85fd-e9d06b2ace25)

**Comment by katinthehatsite** [comment]
Thanks for the screenshot - this helped! I was able to see the iframe: I just needed to add a few buttons first and save them for this preview to appear
